mokapot/config.py

```diff
@@ -328,6 +370,23 @@ def _parser():
         ),
     )
 
+    parser.add_argument(
+        "--suppress_warnings",
+        default=False,
+        action="store_true",
+        help=(
+            "Suppress warning messages when running mokapot. "
+            "Should only be used when running mokapot in production."
+        ),
+    )
+
+    parser.add_argument(
+        "--sqlite_db_path",
+        default=None,
+        type=Path,
+        help="Path of sqlite database to write results directly",
```

Comment:
@gessulat You check that with Vishal?

Reply:
I updated the help text to make clear what this parameter actually does: Optionally it writes results in an sqlite database (which follows our internal result database specification). 

This functionality should probably not live inside  of the Mokapot repo. Vishal said that unfortunately, it will take a little time to factor this out and make it easily hackable. Currently, this parameter is handed down several functions, which makes factoring it out cumbersome. I'll create a separate issue on our side.